composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "bufferapp/php-bufflog",
     "description": "PHP log libraries for Buffer services",
-    "version": "0.1.5",
+    "version": "0.1.6",
```

Comment:
One other thing I thought of - how about including the version number in the log itself in some metadata (eg `['BuffLog_version' => '0.1.6']`)? Not sure how easy that is or if the version number would need to be duplicated. However it would allow us to check for services using old BuffLog versions and nudge teams to upgrade to the latest.

Reply:
Fantastic idea @colinscape . This will help us to keep all services up to date and have a "real" visibility on who use what 💯

Reply:
https://github.com/bufferapp/php-bufflog/issues/22